docs/modules/block.md

```diff
@@ -923,13 +922,13 @@ Play a block sound of the given block.
   **Storage `bs:in block.play_block_sound`**:
   :::{treeview}
   - {nbt}`compound` Block sound data
-    - {nbt}`string` **sound**: The sound to play. Usually took from the `sounds` property of the virtual block (cf get functions).
-    - {nbt}`string` **source**: The source of the sound. Similar to the /playsound command.
-    - {nbt}`string` **targets**: The targets of the sound. Similar to the /playsound command.
-    - {nbt}`string` **pos**: X Y Z coordinates, the position of the sound. Similar to the /playsound command.
-    - {nbt}`int` **volume**: Volume of the sound. Similar to the /playsound command.
-    - {nbt}`int` **pitch**: Pitch of the sound. Similar to the /playsound command.
-    - {nbt}`int` **min_volume**: Minimum volume of the sound. Similar to the /playsound command.
+    - {nbt}`string` **sound**: Sound to play. Found in the `sounds` property of the virtual block (cf get functions).
+    - {nbt}`string` **source**: Source of the sound (default: `master`).
```

Comment:
It can be interesting to keep the reference to the playsound command here, or list all the possibilities.